Mixpanel.podspec

```diff
@@ -14,8 +14,9 @@ Pod::Spec.new do |s|
   s.osx.deployment_target = ''
 
   s.subspec 'Mixpanel' do |ss|
-    ss.source_files  = 'Mixpanel/**/*.{m,h}', 'Mixpanel/**/*.swift'
+    ss.source_files  = 'Mixpanel/**/*.{m,h}', 'Mixpanel/**/*.swift', 'Mixpanel/MPNetwork.{h,m}'
```

Comment:
The recursive check should catch everything. MPNetwork could be removed, but linter was giving me trouble.